### ISSUE TITLE
Pick Name for GEL and rename things

### ISSUE THREAD

**Issue by kylebrandt** [grafana/gel-app#51]
I believe GEL is not going to be the name of this feature. Maybe "Transforms"? 

We should settle on a name and then:
 - [ ] Update the code 
 - [ ] Rename the repo (even if we keep gel, should not be gel-app).

**Comment by bergquist** [comment]
* `grafana-transforms-app`
* `grafana-expressions-app`
* `grafana-metaqueries-app`
* `grafana-gql-app`

ping @ryantxu @torkelo @kylebrandt

**Comment by torkelo** [comment]
> grafana-expressions-app 

btw, is this really an app?

**Comment by ryantxu** [comment]
I like `grafana-expressions-app`

Is it an app... not really, but I don't think we should create a new type for it and the plugin infrastructure is pretty finicky on its naming requirements.

**Comment by kylebrandt** [comment]
> btw, is this really an app?

I don't think so. Soley a backend plugin of a special type. Currently the frontend lives in Grafana.

**Comment by bergquist** [comment]
> Is it an app... not really, but I don't think we should create a new type for it and the plugin infrastructure is pretty finicky on its naming requirements.

:point_up: it has to be panel/datasource/app
I guess datasource could make more sense?

How about 
* `grafana-data-expressions-datasource`
* `grafana-backend-transform-datasource`

**Comment by kylebrandt** [comment]
> it has to be panel/datasource/app

I think this is most like the renderer plugin, which one of those is that?

**Comment by kylebrandt** [comment]
data pipelines or pipeline(s) is another option that could make sense.
